src/cpu/aarch64/acl_winograd_convolution.cpp

```diff
@@ -131,6 +131,15 @@ status_t acl_wino_convolution_fwd_t::pd_t::init_conf() {
 
 status_t acl_wino_convolution_fwd_t::execute_forward(
         const exec_ctx_t &ctx) const {
+    acl_obj_ = std::make_unique<acl_obj_t<Op>>();
+    auto acp = pd()->acp_;
+    acl_obj_->conv.configure(&acp.src_tensor_info, &acp.wei_tensor_info,
+            acp.with_bias ? &acp.bia_tensor_info : nullptr,
+            &acp.dst_tensor_info, acp.padstride_info, acp.act_info,
+            true); // to support 5x5, 7x7 filter shapes in addition to 3x3
+
+    acl_obj_->aux_mem_req = acl_obj_->conv.workspace();
```

Comment:
with the constructor and configure calls, would this impl still be faster?
If not, why not disable the impl from dispatch list?